src/scss/includes/direction.scss

```diff
@@ -287,12 +287,11 @@ button.direction_shortcut {
 
   .itinerary_mobile_step {
```

Comment:
This is the same class you overwrite below in a more specific selector. A lot of the values declared here are overwritten (ex: `position: fixed`) or become obsolete (ex : bottom/left/right).
I suggest you merge both declaration and maybe simplifying the nesting of selectors.